api/server.go

```diff
@@ -112,11 +143,21 @@ func NewServer(ctx context.Context, conf Config) (*Server, error) {
 		}
 	}()
 
+	go func() {
+		s.rpcWebProxy.ListenAndServe()
```

Comment:
You need to handle errors here... see the threads API: https://github.com/textileio/go-threads/blob/master/api/server.go#L84 to line 124